packages/gatsby-theme-carbon/src/util/hooks/useScrollDirection.js

```diff
@@ -9,10 +9,10 @@ const useScrollDirection = () => {
   const windowSize = useWindowSize();
 
   useEffect(() => {
-    const pastPageHeader = y > 240;
-    const isDesktop = windowSize.innerWidth > 1056;
+    const pastPageHeader = y > 290;
+    // const isDesktop = windowSize.innerWidth > 1056;
```

Comment:
am I breaking anything else by removing this @vpicone

Reply:
nah, just gonna clean something up rq